os/drivers/wireless/virtual/vdev_handler.c

```diff
@@ -28,7 +28,7 @@
 #include <time.h>
 #include <tinyara/lwnl/lwnl.h>
 #include <tinyara/net/if/wifi.h>
-#include <tinyara/net/if/wifi_types.h>
+//#include <tinyara/net/if/wifi_types.h>
```

Comment:
Why don't you remove this line?

Reply:
I forgot to remove the line. I'll delete the line